docs/storybook/src/AppUiStory.tsx

```diff
@@ -187,6 +182,65 @@ export function Page() {
   );
 }
 
+/**
+ * A simplified App UI story wrapper for stories that only need frontstages and itemProviders
+ * without performing the full IModelApp/UIFramework startup. This assumes the environment
+ * (IModelApp/UiFramework) is already initialized by a decorator (e.g., `InitializerDecorator`).
+ */
+export function SimpleAppUiStory(props: {
```

Comment:
Why is this needed? Take a look at other stories and stay consistent.

Reply:
removed